java/pom.xml

```diff
@@ -41,6 +41,7 @@
     <arrow.vector.classifier></arrow.vector.classifier>
     <forkCount>2</forkCount>
     <checkstyle.failOnViolation>true</checkstyle.failOnViolation>
+    <javac.version>9+181-r4173-1</javac.version>
```

Comment:
ok i think this is the errorprone's javac version? if yes can we name the property as such :)

Reply:
Right, fixed.